transformer/catalog-transformer-xml/src/main/java/ddf/catalog/transformer/xml/XmlResponseQueueTransformer.java

```diff
@@ -54,52 +366,80 @@
             LOGGER.info("Failure creating MIME type", e);
             throw new ExceptionInInitializerError(e);
         }
+
+        TYPE_NAME_LOOKUP = new ImmutableMap.Builder<AttributeType.AttributeFormat, String>()
+                .put(AttributeFormat.BINARY, "base64Binary").put(AttributeFormat.STRING, "string")
+                .put(AttributeFormat.BOOLEAN, "boolean").put(AttributeFormat.DATE, "dateTime")
+                .put(AttributeFormat.DOUBLE, "double").put(AttributeFormat.SHORT, "short")
+                .put(AttributeFormat.INTEGER, "int").put(AttributeFormat.LONG, "long")
+                .put(AttributeFormat.FLOAT, "float").put(AttributeFormat.GEOMETRY, "geometry")
+                .put(AttributeFormat.XML, "stringxml").put(AttributeFormat.OBJECT, "object")
+                .build();
+
+        String nsPrefix = "xmlns";
+
+        NAMESPACE_MAP = new ImmutableMap.Builder<String, String>()
+                .put(nsPrefix, "urn:catalog:metacard")
+                .put(nsPrefix + ":ns" + 1, "http://www.opengis.net/gml")
```

Comment:
you could give these more meaningful namespaces to help with the deduplication. `http://www.opengis.net/gml` could be `gml` instead of `ns1`.